base/Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM golang:1.12.9-stretch AS builder
+
+MAINTAINER free5GC <support@free5gc.org>
+
+ENV GO111MODULE=off
+
+RUN apt-get update
+RUN apt-get -y install gcc cmake autoconf libtool pkg-config libmnl-dev libyaml-dev
+RUN apt-get -y install netcat tcpdump iproute2 netbase
```

Comment:
netcat, tcpdump, iproute2 is debug tools.

I have check you have debug tool on each image, so I think we don't need to install here.

Reply:
I think netbase also can be removed since it will be added later to the n3iwf dependencies.

Reply:
Yes, it's just a dependency of n3iwf